go/vt/mysqlctl/metadata_tables.go

```diff
@@ -84,13 +109,30 @@ func PopulateMetadataTables(mysqld MysqlDaemon, localMetadata map[string]string,
 		return err
 	}
 
-	// Create the database and table if necessary.
+	return createMetadataTables(conn, dbName)
+}
+
+func createMetadataTables(conn *dbconnpool.DBConnection, dbName string) error {
```

Comment:
... following up on my previous comment, we now have one method `CreateMetadataTables`, one function `CreateMetadataTables`, one function `createMetadataTables`. Let's try and make these less ambiguous by renaming the latter two?

Reply:
Yeah, how about this:

* `MetadataManager` has public methods: `{Create,Populate}MetadataTables` and `UpsertMetadataTables`
* The main reason I did this "public function and also public method on struct" thing was to reduce the number of changes I needed to do (because otherwise I would have to update everyone that called `mysqlctl.PopulateMetadataTables` .... which now that i look is just two places in `mysqlctl/backup.go` so maybe I should just change them 🤔 )
* The other reason for the public/private with the same name was to allow code reuse without having to get new dba connections for each stage, so in `Populate`, we first get a connection, and then call private `create` and private `upsert`, whereas if we called public `Create` and public `Upsert` then we would get a new connection for each of those.

Then, we do:

```go
type MetadataManager struct {}

func (m *MetadataManager) CreateMetadataTables(mysqld MysqldDaemon, dbName string) error {
    log.Infof("....")
    conn, err := mysqld.GetDbaConnection(...)

Reply:
Sounds good. We can do so iteratively in followup PRs.